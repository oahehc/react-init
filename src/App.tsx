import React from "react";
import styled from 'styled-components'

const Title = styled.h1`
  font-size: 1.5em;
  text-align: center;
  color: palevioletred;
`;

const App = () => {
  return (
    <div>
      <Title>Test a little.</Title>
    </div>
  );
};

export default App;