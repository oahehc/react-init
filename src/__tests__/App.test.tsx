import React from "react";
import { mount } from "enzyme";
import App from "../App";

describe("App", () => {
  it("should match the snapshot", () => {
    const subject = mount(<App />);

    expect(subject).toMatchSnapshot();
  });

  it("should have Title", () => {
    const subject = mount(<App />);

    expect(subject.find("App__Title")).toHaveLength(1);
  });
});
