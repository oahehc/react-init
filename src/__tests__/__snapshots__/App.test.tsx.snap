// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should match the snapshot 1`] = `
<App>
  <div>
    <App__Title>
      <h1
        className="App__Title-sc-10m2fy2-0 kfYYTB"
      >
        Test a little.
      </h1>
    </App__Title>
  </div>
</App>
`;
